pycaching/geocaching.py

```diff
@@ -234,7 +234,7 @@ def search(self, point, limit=float("inf")):
                 c = Cache(self, wp)
                 c.type = Type.from_string(cache_details[0].strip())
                 c.name = row.find("span", "cache-name").text
-                c.found = row.find("img", title="Found It!") is not None
+                c.found = str(row.find("svg", class_="badge")).find('found') >= 0
```

Comment:
Hello @ian-howell, thanks for your first PR to pycaching. :-)

But this is quite a hack for more that one reason. :-) The `row.find("svg", class_="badge")` part returns a beautiful soup element or `None`, depending on the lookup result. So far so good. But the `str(...)` part converts it to string, even if `None` is returned. So the code does something like `"None".find('found') >= 0`, which is really not ideal, regardless a fact, that prefered way of searching for substrings is using `in` operator: `"found" in "None"`.. :-)

So please can you split this line to 2 steps? 1st – finding the element, 2nd – finding `"found"` substring *only if* the element was found previously? And please, use the double quotes to match the project's code style. ;-)

Thank you! 👍